CONTRIBUTING.md

```diff
@@ -51,6 +51,13 @@ Once the pull request is approved, a team member will take care of merging.
 
 Python 3.7 or later is required.
 
+For Windows, before setting up development environment, make sure you have WSL 
```

Comment:
Thanks! Let's make this it's own section

Developing on Windows

And bring the bit below about line ending for WSL into the section as well.